src/pages/contactUs/atoms/Label.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components'
+
+export const Label = styled.label`
+  color: ${({ theme }) => theme.colors.blackLight};
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 28px;
```

Comment:
You can also use plain numbers (i.e. `1.5`) that would be calculated from the current element's `font-size`:

```css
font-size: 16px;
line-height: 1.5; // 1.5 * 16px = 24px
```

Reply:
is my option wrong?

Reply:
Nope, just a suggestion :)